test/e2e/specs/demo.test.js

```diff
@@ -1,42 +1,39 @@
+/**
+ * External dependencies
+ */
+import fetch from 'node-fetch';
+
 /**
  * Internal dependencies
  */
 import { visitAdmin } from '../support/utils';
 
-// The response from Vimeo, but with the iframe taken out.
-const MOCK_EMBED_RESPONSE = {
-	type: 'video',
-	version: '1.0',
-	provider_name: 'Vimeo',
-	provider_url: 'https://vimeo.com/',
-	title: 'The Mountain',
-	author_name: 'TSO Photography',
-	author_url: 'https://vimeo.com/terjes',
-	is_plus: '0',
-	account_type: 'basic',
-	html: '<p>Embedded content</p>',
```

Comment:
Thank you for your explanation around embeds @notnownikki 👍  I think I understand now the reason for this PR. There is just one thing I'm missing. Would the test be valid if we hardcoded the Vimeo response without the src as the HTML in this line?

Reply:
No, because it would always pass, even if Vimeo's response changed and caused the post to be dirty in real world use. If Vimeo decide to change their iframe dimensions for some reason, then in actual use the post might end up dirty, but in the test it would pass.

This seems like an unlikely thing to happen, but YouTube did it not too long ago with 16:9 and 4:3 videos getting different aspect ratios, so I don't want to run the risk of us having a test that says "Hey, everything is fine!" and causing us to miss actual errors.